docs/source/reference/rest_api.rst

```diff
@@ -946,49 +995,246 @@ All of them must be followed by the operator ``=``.
             http://localhost:5000/api/v4/computers?orderby=+name
             http://localhost:5000/api/v4/computers?orderby=-uuid
 
-    :attributes_filter:
-        This key is used to specify which attributes of a specific object have to be returned.
-        The desired attributes have to be provided as a comma-separated list of values.
-        It is used in the endpoints ``/contents/attributes`` and ``/nodes``.
+    * - ``attributes_filter``
+      - A comma-separated list of attributes to return.
+        Use together with ``attributes=true``.
+        Available in the endpoints ``/contents/attributes`` and ``/nodes``.
         Example:
 
         ::
 
             http://localhost:5000/api/v4/nodes/4fb10ef1/contents/attributes?attributes_filter=append_text,prepend_text
 
-    :extras_filter: Similar to ``attributes_filter`` but for extras. It is used in the endpoints ``/contents/extras`` and ``/nodes``.
+    * - ``extras_filter``
+      - Similar to ``attributes_filter`` but for extras. It is used in the endpoints ``/contents/extras`` and ``/nodes``.
+
+    * - ``attributes``
+      - Pass ``true`` in order to return attributes in the ``/nodes`` endpoint (excluded by default).
+
+    * - ``extras``
+      - Pass ``true`` in order to return extras in the ``/nodes`` endpoint (excluded by default).
+
+    * - ``download_format``
+      - to specify download format in ``/download`` endpoint.
+
+    * - ``download``
+      - in ``/download`` endpoint, if ``download=false`` it displays the content in the browser instead of downloading a file.
+
+    * - ``filename``
+      - this filter is used to pass file name in ``/repo/list`` and ``/repo/contents`` endpoint.
+
+    * - ``tree_in_limit``
+      - specifies the limit on tree incoming nodes.
+
+    * - ``tree_out_limit``
+      - specifies the limit on tree outgoing nodes.
+
+
+Regular filters can be compounded, requiring all specified filters to apply.
+
+
+.. list-table:: Regular filters
```

Comment:
Could you sort these (and the one above) alphabetically perhaps?

Reply:
done!